backend/tests/integration/test_account.py

```diff
@@ -195,6 +202,161 @@ def test_invalid_account_update(self, mock_verify_oauth2_token):
         )
 
 
+@mock.patch("google.oauth2.id_token.verify_oauth2_token")
+class CreatePrivacyTests(TransactionTestCase):
+    reset_sequences = True
+
+    def setUp(self):
+        self.client.cookies = SimpleCookie({"csrftoken": "valid_csrf_token"})
+        self.header = {
+            "ACCEPT": "application/json",
+            "HTTP_X-CSRFToken": "valid_csrf_token",
+        }
+
+    def test_get_privacies(self, mock_verify_oauth2_token):
+        mock_verify_oauth2_token.return_value = {
+            "sub": "unique_user_id",
+            "given_name": "firstname",
+            "email": "useremail",
+        }
+        # Create an account first
+        response = self.client.post(
```

Comment:
I'd suggest using `UserFactory` rather than creating a user manually in Privacy integration tests (like is done in `test_job` integration tests) - this also allows removal of all csrf related mocking. Open to other opinions, but probably more readable to use factories for non-privacy models.

Reply:
Privacies are auto-created as part of creating a user via the views->business interactions, so I would like to keep these as-is, even if the csrf thing is a bit much. I can comment some more to clarify things?

Reply:
Ok yeah that's probably fine, just adding something to your comment on line 222 that `get_or_create_account` creates Privacy should be enough

Reply:
Ok yeah that's probably fine, just adding something to your comment on line 222 that `get_or_create_account` creates Privacy should be enough